doc/source/_templates/layout.html

```diff
@@ -1,4 +1,8 @@
 {% extends "!layout.html" %}
 
-{% block relbar2 %}{% endblock %}
+{% block extrahead %}
+<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
```

Comment:
This is probably not a huge issue for expyfun, but FYI any website that includes calls to `*.googleapis.com/*` will hang indefinitely and never finish loading if accessed from China (without a VPN). There are also issues with some configurations of Apache servers not allowing cross-domain imports via the `link` tag.  As much as google would prefer you to use their hosted version of the font, I slightly prefer hosting font & website on the same server.  Feel free to say "won't fix", just wanted you to know about it.